Crash when attempting to get device details

#### Software and hardware versions 
Collect v1.4.10+, seen across devices and Android versions in Play Store console

#### Problem description
Collect crashes when trying to get the phone's serial number. I believe `telephonyManager.getDeviceId()` is the culprit.

#### Steps to reproduce the problem
I'm not able to reproduce this on my devices. Galaxy Note5, Galaxy S6 are common devices this happens on.

#### Expected behavior
No crash. The calls that can result in exceptions should be wrapped in try/catch blocks.

#### Other information 
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5973)
  at android.app.ActivityThread.-wrap3(ActivityThread.java:0)
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1710)
  at android.os.Handler.dispatchMessage(Handler.java:102)
  at android.os.Looper.loop(Looper.java:154)
  at android.app.ActivityThread.main(ActivityThread.java:6776)
  at java.lang.reflect.Method.invoke(Native Method:0)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1496)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1386)
Caused by: java.lang.SecurityException: 
  at android.os.Parcel.readException(Parcel.java:1693)
  at android.os.Parcel.readException(Parcel.java:1646)
  at com.android.internal.telephony.IPhoneSubInfo$Stub$Proxy.getIccSerialNumberForSubscriber(IPhoneSubInfo.java:910)
  at android.telephony.TelephonyManager.getSimSerialNumber(TelephonyManager.java:2493)
  at android.telephony.TelephonyManager.getSimSerialNumber(TelephonyManager.java:2476)
  at org.odk.collect.android.logic.PropertyManager.<init>(PropertyManager.java:118)
  at org.odk.collect.android.application.Collect.onCreate(Collect.java:253)
  at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1032)
  at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5970)
```

--- a/collect_app/src/main/java/org/odk/collect/android/logic/PropertyManager.java
+++ b/collect_app/src/main/java/org/odk/collect/android/logic/PropertyManager.java
@@ -80,19 +80,23 @@ public class PropertyManager implements IPropertyManager {
     public PropertyManager(Context context) {
         Timber.i("calling constructor");
 
-        // Device-defined properties
-        TelephonyManager telMgr = (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
-        IdAndPrefix idp = findDeviceId(context, telMgr);
-        putProperty(PROPMGR_DEVICE_ID,     idp.prefix,          idp.id);
-        putProperty(PROPMGR_PHONE_NUMBER,  SCHEME_TEL,          telMgr.getLine1Number());
-        putProperty(PROPMGR_SUBSCRIBER_ID, SCHEME_IMSI,         telMgr.getSubscriberId());
-        putProperty(PROPMGR_SIM_SERIAL,    SCHEME_SIMSERIAL,    telMgr.getSimSerialNumber());
+        try {
+            // Device-defined properties
+            TelephonyManager telMgr = (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
+            IdAndPrefix idp = findDeviceId(context, telMgr);
+            putProperty(PROPMGR_DEVICE_ID,     idp.prefix,          idp.id);
+            putProperty(PROPMGR_PHONE_NUMBER,  SCHEME_TEL,          telMgr.getLine1Number());
+            putProperty(PROPMGR_SUBSCRIBER_ID, SCHEME_IMSI,         telMgr.getSubscriberId());
+            putProperty(PROPMGR_SIM_SERIAL,    SCHEME_SIMSERIAL,    telMgr.getSimSerialNumber());
 
-        // User-defined properties. Will replace any above with the same PROPMGR_ name.
-        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
-        initUserDefined(prefs, KEY_METADATA_USERNAME,    PROPMGR_USERNAME,      SCHEME_USERNAME);
-        initUserDefined(prefs, KEY_METADATA_PHONENUMBER, PROPMGR_PHONE_NUMBER,  SCHEME_TEL);
-        initUserDefined(prefs, KEY_METADATA_EMAIL,       PROPMGR_EMAIL,         SCHEME_MAILTO);
+            // User-defined properties. Will replace any above with the same PROPMGR_ name.
+            SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+            initUserDefined(prefs, KEY_METADATA_USERNAME,    PROPMGR_USERNAME,      SCHEME_USERNAME);
+            initUserDefined(prefs, KEY_METADATA_PHONENUMBER, PROPMGR_PHONE_NUMBER,  SCHEME_TEL);
+            initUserDefined(prefs, KEY_METADATA_EMAIL,       PROPMGR_EMAIL,         SCHEME_MAILTO);
+        } catch (SecurityException e) {
+            Timber.e(e);
+        }
     }
 
     private IdAndPrefix findDeviceId(Context context, TelephonyManager telephonyManager) {

